proto/axelar/evm/v1beta1/query.proto

```diff
@@ -191,3 +192,17 @@ message TokenInfoResponse {
   bool is_external = 5;
   string burner_code_hash = 6;
 }
+
+message Proof {
+  repeated bytes operators = 1
```

Comment:
`operators`, `weight`, and `threshold` should all of type string to avoid the caller having the deserialize go types.